### DISCUSSION TITLE
Suggestion - projected load should reflect known distorting events like Saving Sessions and Power-ups

### DISCUSSION THREAD

**Discussion by PianSom** [springfall2008/batpred#713] [Ideas]
Anecdotally, occasional Octopus events that Predbat is aware of (I'm thinking of Saving Sessions and Power-ups/rates_import_overide) change the consumption behaviour from "normal". (That's what they are supposed to do, anyway!)

Might it be possible for the projected load calculation, which currently just reflects a  fixed historic usage averaging process, to be amended to exclude those days which have had such an event from inclusion in the calculation?

And is this actually a worthwhile change?

**Comment by gcoan** [comment]
There is the ability to change the predicted load for a power up event and a saving session, so think this is covered now
